components/styles/readme.md

```diff
@@ -0,0 +1,172 @@
+Stack Component
+===============
+
+The Stack Component is a powerful, reusable SCSS tool built upon the robust CSS Grid framework. It's designed to manage both layout and spacing, thus promoting uniformity, readability, and maintainability within your CSS code.
```

Comment:
it's not scss tool, it's a react component and not built with grid, rather with flexbox